docs/orca_data_definitions.md

```diff
@@ -84,6 +85,26 @@ A given file may contain file paths to be updated for use on the grading VM. For
 
 Grading jobs have a numeric priority determined by Bottlenose, which is interpreted as a _delay_ to be applied to the job when added to the queue.
 
+Jobs are run inside Docker containers to provide a level of isolation from the local machine.
+
+<hr>
+
+### `GraderImageConfig`
+
+```typescript
+interface GraderImageConfig {
+  repository: string;
```

Comment:
This should only ever be a singlle SHA. This avoids naming collisions and circumvents versioning complexity on the part of professors who create images.